src/app/_components/PortalComponents/Portal/users/UsersTable.tsx

```
@@ -1,39 +1,40 @@
+  const { user } = useUser();
+  const handleChange = (userId: string, role: UserRole) => {
+    mutateUserRole.mutate({ userId, role });
   };
-
   const userColumns = columns(user, handleChange);
 
   return (
     <EntityTable
-      columns={userColumns as ColumnDef<User, unknown>[]}
       data={data}
+      columns={userColumns as ColumnDef<User, unknown>[]}
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)